konnectd/ui/src/containers/Login/Chooseaccount.js

```diff
@@ -33,7 +33,7 @@ const styles = theme => ({
 });
 
 function Chooseaccount(props) {
```

Comment:
It just saves one line, but we could have the signature like this:
`function Chooseaccount({ loading, errors, classes, hello, history, dispatch }) {`